### ISSUE TITLE
Finetuning setting of Tune-A-Video

### ISSUE THREAD

**Issue by zhouliang-yu** [seervideodiffusion/seervideodiffusion.github.io#2]
Hi Xianfan

I have some questions related to the finetuning setting of Tune-A-Video baseline you mentioned in your paper, I adopt a similar training setting with seer.

1. Parse the sequential images to the latent
2. keep the first frame latent unchanged, and add noise to the other latents
3. denoising the noisy latents with U-net 

When inferencing, I parse the initial frame to latent, the conditional latent vectors are concatenated with Gaussian noise vectors, and feed them to the Tune-A-Video pipeline. But I get a really bad result, it can reconstruct the first frame, but the other frames are very noisy, even the training loss is small. 

could you pls share the code for finetuning the tune-a-video baseline?

My email is zhouliangyu@link.cuhk.edu.cn
